packages/manager/modules/netapp/src/dashboard/routing.js

```diff
@@ -1,14 +1,34 @@
+import NetApp from './Netapp.class';
+
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('netapp.dashboard', {
     url: '/:serviceName',
-    component: 'ovhManagerNetAppDashboard',
+    views: {
+      netappContainer: {
+        component: 'ovhManagerNetAppDashboard',
+      },
+    },
     resolve: {
       currentActiveLink: /* @ngInject */ ($transition$, $state) => () =>
         $state.href($state.current.name, $transition$.params()),
-      dashboardLink: /* @ngInject */ ($state) =>
-        $state.href('netapp.dashboard'),
+      dashboardLink: /* @ngInject */ ($state, $transition$) =>
+        $state.href('netapp.dashboard', $transition$.params()),
+      createVolumeLink: /* @ngInject */ ($state, $transition$) =>
+        $state.href('netapp.dashboard.volume.create', $transition$.params()),
```

Comment:
State `netapp.dashboard.volumes` will be corresponding to Volumes tab, where `netapp.dashboard.volume` will be the volume detail, so:
```suggestion
        $state.href('netapp.dashboard.volumes.create', $transition$.params()),
```